src/generator/AutoRest.Ruby/Model/RequirementsRb.cs

```diff
@@ -184,11 +152,11 @@ public virtual string GetDependencyGems()
 require 'concurrent'
 require 'ms_rest'";
 
-            if(!string.IsNullOrWhiteSpace(this.ns))
+            if(!string.IsNullOrWhiteSpace(Settings.Instance.Namespace))
             {
                 requirements = requirements 
                     + Environment.NewLine 
-                    + string.Format(CultureInfo.InvariantCulture, "require '{0}/{1}/module_definition'", GeneratedFolderName, this.sdkName);
+                    + string.Format(CultureInfo.InvariantCulture, "require '{0}/{1}/module_definition'", GeneratedFolderName, GeneratorSettingsRb.Instance.sdkName);
```

Comment:
>GeneratorSettingsRb.Instance.sdkName) [](start = 126, length = 37)

I am noticing different behavior for this value after generation. 

It supposed to generate a line like `require 'generated/azure_mgmt_authorization/module_definition'` but it's not generating line as `require 'generated/authorization/module_definition'`

Did you notice that?